RiotSwiftUI/Modules/Room/LocationSharing/LocationSharingModels.swift

```diff
@@ -19,14 +19,23 @@ import SwiftUI
 import Combine
 import CoreLocation
 
+// This is the equivalent of MXEventAssetType in the MatrixSDK
+enum LocationSharingCoordinateType {
```

Comment:
You can make just 2 cases in the app side. `case pin, case user`. As you will not handle generic case in your screens.